samples/server/petstore/go-api-server/go/api.go

```diff
@@ -57,14 +58,14 @@ type UserApiRouter interface {
 // while the service implementation can ignored with the .openapi-generator-ignore file 
 // and updated with the logic required for the API.
 type PetApiServicer interface { 
-	AddPet(Pet) (interface{}, error)
-	DeletePet(int64, string) (interface{}, error)
-	FindPetsByStatus([]string) (interface{}, error)
-	FindPetsByTags([]string) (interface{}, error)
-	GetPetById(int64) (interface{}, error)
-	UpdatePet(Pet) (interface{}, error)
-	UpdatePetWithForm(int64, string, string) (interface{}, error)
-	UploadFile(int64, string, *os.File) (interface{}, error)
+	AddPet(context.Context, Pet) (interface{}, int, error)
+	DeletePet(context.Context, int64, string) (interface{}, int, error)
+	FindPetsByStatus(context.Context, []string) (interface{}, int, error)
+	FindPetsByTags(context.Context, []string) (interface{}, int, error)
+	GetPetById(context.Context, int64) (interface{}, int, error)
+	UpdatePet(context.Context, Pet) (interface{}, int, error)
+	UpdatePetWithForm(context.Context, int64, string, string) (interface{}, int, error)
+	UploadFile(context.Context, int64, string, *os.File) (interface{}, int, error)
```

Comment:
@randomswdev thanks for the PR. Is it correct to say that this PR introduces breaking changes as the method signature has changed?

Reply:
Yes, this merge request changes the signature of the generated methods. While the changes are breaking ones, I think that adding the context propagation is much needed to allow generating really useful go stubs.